pages/Arley-AV/index.html

```diff
@@ -0,0 +1,388 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jesus Arley Valencia Tejada</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="slider">
+      <div class="slider__container">
+        <h1 class="slider__title">
+          Our <span class="slider__title--orange">Hotels</span>
+        </h1>
+        <div class="slider__cards">
+          <div class="card">
+            <div class="card__container">
+              <div class="card__image">
+                <img src="images/download.jpeg" alt="Cuba - Beach" />
+              </div>
+              <div class="card__info">
+                <h3 class="card__title">
+                  Cuba
+                </h3>
+                <ul class="card__locations">
+                  <li class="location">
+                    <span class="location__name"
+                      >Mella Santiago de Cuba img</span
+                    >
+                    <div class="location__stars-rating">
+                      <span class="stars-rating__star"
```

Comment:
this class could be better named, remember: Do you think `starts-rating` could be a separate element?